inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -230,15 +230,49 @@ def simulate(
         resource_pool_id: typing.Optional[uuid.UUID] = None,
         run_async: bool = False,
         plasma_surface_filepath: typing.Optional[types.Path] = None,
+        num_iterations: int = None,
+        num_samples: int = None,
+        sigma_scaling_factor: float = None,
     ) -> tasks.Task:
         """Simulates the scenario.
 
+        A simulation can be performed in two ways simply by adding some more
+        arguments. 
+        
+        If `num_iterations`, `num_samples` and `sigma_scaling_factor`
+        are all not provided, the simulation merely uses the created device and 
+        performs the computation of the magnetic field produced by its coils
+        on the given plasma surface. Additionally, it produces an output of
+        the objective functions that assess the quality of the device.
```

Comment:
A simpler, more natural way to provide this functionality is by setting defaults to these parameters, namely:
```python
num_samples = 1
num_iterations = 1
sigma_scaling_factor = 0.1
```

Then explain what happens when `num_samples` and `num_iterations` > 1. WDYT?

Reply:
I totally agree, much simpler and much easier to understand, thank you!